File: C_Programming-main/C_College/C Code/Calculator.c
#include<stdio.h>
#include<conio.h>
void main()
{
    int num1,num2,result;
    char op;
    printf("\t\t.............Welcome To Shield Squads CALCULATOR Program............");
    printf("\n\nEnter the first Number : ");
    scanf("%d",&num1);
    printf("Enter the Operator : ");
    scanf(" %c",&op);
    printf("Enter the second Number : ");
    scanf("%d",&num2);
    switch(op)
    {
        case '+':
            result=num1+num2;
            printf("\t\t\t\tAddition");
            printf("\nSum of  %d and %d is : %d",num1,num2,result);
        break;
        case '-':
            result=num1-num2;
            printf("\t\t\t\tSubtraction");
            printf("\nDifferences of  %d and %d is : %d",num1,num2,result);
        break;
        case '*':
            result=num1*num2;
            printf("\t\t\t\tMultiplication");
            printf("\nMultiplication of  %d and %d is : %d",num1,num2,result);
        break;
        case '/':
            result=num1/num2;
            printf("\t\t\t\tDivision");
            printf("\nDivision of  %d and %d is : %d",num1,num2,result);
        break;
        default :
            printf("\t\t\tThe operator is not valid");
    }
    getch();
}

File: DSA_Practice-main/Module 7/calculator.java
import java.util.Scanner;

public class calculator {
     public static void main(String[] args) {
          Scanner sc = new Scanner(System.in);

          System.out.print("Enter the 1st Number: ");
          float num1 = sc.nextFloat();
          System.out.print("Enter the 2nd Number: ");
          float num2 = sc.nextFloat();

          System.out.print("ENter the Sign(+, -, *, /, %): ");
          String sign = sc.next();

          switch (sign) {
               case "+":
                    System.out.println(num1 + " + " + num2 + " = " + (num1 + num2));
                    break;
               case "-":
                    System.out.println(num1 + " - " + num2 + " = " + (num1 - num2));
                    break;
               case "*":
                    System.out.println(num1 + " x " + num2 + " = " + (num1 * num2));
                    break;
               case "/":
                    System.out.println(num1 + " / " + num2 + " = " + (num1 / num2));
                    break;
               case "%":
                    System.out.println(num1 + " % " + num2 + " = " + (num1 % num2));
                    break;
               default:
                    System.out.println("Please enter valid sign!");
                    break;
          }
     }
}
